src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -61,11 +61,10 @@ public String saveItem(Item item) throws Exception {
      * 400: item not found
      *
      * @param id
-     * @return
+     * @return ResponseEntity(HttpStatus.OK)
```

Comment:
Good... though I'd just say '200', and describe when.
What about exception cases?